services/blockchain-indexer/shared/dataService/business/interoperability/blockchainApps.js

```diff
@@ -71,11 +75,30 @@ const getBlockchainApps = async (params) => {
 
 	const total = await blockchainAppsTable.count(params);
 
-	blockchainAppsInfo.data = await blockchainAppsTable.find(
+	const resultSet = await blockchainAppsTable.find(
```

Comment:
Would something indicating db values be a more suitable name. In the below loop it felt a bit confusing to loop over `resultSet` calling each item as `blockchainAppInfo` and assigning the array back to `blockchainAppInfo.data`. Maybe we can rename this to something like `dbBlockchainApps` or something more related?